dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/LogsFeatureTest.kt

```diff
@@ -196,6 +202,49 @@ internal class LogsFeatureTest {
         assertThat(endpointUrl).isSameAs(endpointUrl2)
     }
 
+    @Test
+    fun `it will register and unregister the provided plugin when initialise and stop called`(
```

Comment:
Just to be safe, I'd split this in 2 tests : verifying that initialize only calls register, and stop only calls unregsiter.